packages/react/src/Prose/Prose.module.css

```
@@ -0,0 +1,205 @@
+/* ---------------------------------------------------------- */
+/* Prose component                                            */
+/* ---------------------------------------------------------- */
+/* Table of contents:                                         */
+/* 1. Container                                               */
+/* 2. Element spacing                                         */
+/* 3. Line-length control                                     */
+/* 4. Headings                                                */
+/* 5. Paragraphs                                              */
+/* 6. Lists                                                   */
+/* 7. Inline elements                                         */
+/* 8. Images                                                  */
+/* ---------------------------------------------------------- */
+
+/* ---------------------------------------------------------- */
+/* 1, Container                                               */
+/* ---------------------------------------------------------- */
+
+.Prose {
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-300);
+  line-height: var(--brand-text-lineHeight-300);
+  color: var(--brand-color-text-default);
+}
+
+/* ---------------------------------------------------------- */
+/* 2. Element spacing                                         */
+/* ---------------------------------------------------------- */
+
+.Prose * + * {
+  margin-block-start: var(--spacing, 1em);
```

Could we use rem instead of em as the default value?

```suggestion
  margin-block-start: var(--spacing, 1rem);
```

---

Using rem wouldn't have the intended effect. The spacing should be relative to the element's font-size, not the root element's font-size. We want larger elements to have larger spacing. If we used `rem` all elements would have the same spacing.

---

This has been reverted to use em